src/test/optimizer/join_graph_builder_test.cpp

```diff
@@ -309,4 +312,25 @@ TEST_F(JoinGraphBuilderTest, BuildAllInLQP) {
   EXPECT_EQ(join_graphs.at(1).vertices.at(1), node_b);
 }
 
+TEST_F(JoinGraphBuilderTest, MultiPredicateJoin) {
+  const std::vector<std::shared_ptr<AbstractExpression>> join_predicates{equals_(a_a, b_a), greater_than_(a_b, b_b),
+                                                                         less_than_equals_(a_c, b_c)};
+  const auto full_outer_join_node = JoinNode::make(JoinMode::FullOuter, join_predicates, node_a, node_b);
+  const auto lqp = JoinNode::make(JoinMode::Inner, join_predicates, node_a, full_outer_join_node);
```

Comment:
pls build LQPs as follows:

```c++
// clang-format off
const auto lqp = 
JoinNode::make(JoinMode::Inner, join_predicates, 
  node_a, 
  JoinNode::make(JoinMode::FullOuter, join_predicates, 
    node_a, 
    node_b));
// clang-format on
```

to keep consistency with other places where LQPs are built